feat: updates to TODO, small updates to chapter 3 and chapter 5 of bg (#971)
flybywiresim/docs: docs/pilots-corner/a380x/a380x-beginner-guide/05_takeoff-climb-cruise.md

<link rel="stylesheet" href="/stylesheets/bg.css">

# Takeoff, Climb and Cruise

This guide will explain the correct procedures to accomplish takeoff, climb and establish cruise altitude.

!!! warning "Disclaimer"

>     The level of detail in this guide is meant to get a FlyByWire A380X beginner safely up in the air and to cruise level under normal conditions, while simplifying details which are not (yet) important for a beginner.

Reviewer: Be careful were we mean the actual aircraft (A320neo or A380) and were we mean our specific project aircraft (A32NX and A380X)

```suggestion
    The level of detail in this guide is meant to get a FlyByWire A380 beginner safely up in the air and to cruise level under normal conditions, while simplifying details which are not (yet) important for a beginner.
```

    A *beginner* is defined as someone familiar with flying a GA aircraft or different types of airliners. Aviation terminology and know-how is a requirement to fly any airliner, even in Microsoft Flight Simulator.

    

---

??? tip "Quick Start"
    MSFS Start from Gate or Runways

    Microsoft Flight Simulator allows you to start your flight from cold & dark at a gate or directly from the runway with the aircraft ready for takeoff.
    For this guide, we assume you started cold & dark at the gate and taxied to the runway holding point as per the previous chapters of this beginner guide.
    If you did start on the runway, you can skip the first part (Lineup) and directly continue reading [Takeoff](#2-takeoff).

## Prerequisites

> Aircraft is in TAXI state as per previous chapters

Reviewer: ```suggestion
Aircraft is in TAXI state as per previous chapters.
```

[Download FlyByWire Checklist](../assets/sop/A32NX%20Documentation/FBW%20A32NX%20Checklist.pdf){ .md-button } <p style="color:yellow; font-size:18px;">TODO: update checklist</p>

## Chapters / Phases

> This guide will cover these phases: 

Reviewer: make sure your editor removes whitespaces at the end of lines on save - otherwise we get these false differences

---

## Base Knowledge About the Airbus A380 for Flight

> This list is focussed on differences to other non-Airbus airliners a user might be used to.

Reviewer: ```suggestion
This section is focused on differences to other non-Airbus airliners a user might be used to.
```

> ??? tip "What and Why"

Reviewer: ```suggestion
??? tip "What is a Fly-by-Wire System?"
```

>     Fly By Wire system

Reviewer: ```suggestion
```

    In fly-by-wire systems, when the pilot moves flight controls, those movements are converted into electronic signals, which are then interpreted by the aircraft's Electrical Flight Control System (EFCS) to adjust actuators that move flight control surfaces. Computers also monitor sensors throughout the aircraft to make automatic adjustments that enhance the flight.

    Because fly-by-wire is electronic, it is much lighter and less bulky than mechanical controls, allowing increases in fuel efficiency and aircraft design flexibility, even in legacy aircraft. And to prevent flight critical failure, most fly-by-wire systems also have triple or quadruple redundancy back-ups built into them. [source: BAE Systems](https://www.baesystems.com/en-us/definition/what-are-fly-by-wire-systems){target=new}

     See also: [Fly-by-wire Wikipedia](https://en.wikipedia.org/wiki/Fly-by-wire){target=new}

> ??? tip "Specific Airplane Flight Systems" 

Reviewer: I would rather split these up - like this:

```suggestion
??? tip "What is Autotrim?" 
```

    Autotrim:
    The A380 has a feature called "Autotrim", which makes it unnecessary to hold the sidestick or use the trim wheel for holding the current pitch. This system is always active, even when the __Autopilot__ is off (in Normal Law, which means under normal circumstances with a fully functional aircraft).

>     Autothrust:

Reviewer: separate collapsible "What is Autothrust?"

    The A380 has __Autothrust__ which is similar to Autothrottle (e.g., in a Boeing), but it does not move the thrust levers. Basically, the thrust levers are only moved by the pilot and never move on their own. The thrust levers act as a maximum allowed power setting for the Autothrust system. During normal flight (after takeoff) the levers stay in the CL climb detent, and the Autothrust system will set engine power accordingly.

>     Autopilot:

Reviewer: separate collapsible "Airbus Autopilot"

    The A380's __Autopilot__ system works a bit differently from other manufacturer's systems. The A380 FCU controls allow setting certain values and then push or pull the knobs. Pushing usually means automatic control (Managed Mode) and pulling will use the manually selected value (Selected Mode).

    ![FCU](../assets/beginner-guide/takeoff-climb-cruise/FCU.png "FCU"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot with new FCU</p>

    !!!info "Microsoft Flight Simulator knobs"
        In Microsoft Flight Simulator, pushing is clicking the knob "UP" and pulling is clicking the knob "DOWN"<br/>
        ![Pulling a knob in MSFS](../assets/beginner-guide/takeoff-climb-cruise/Pulling-knob.png "Pulling a knob in MSFS"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot with new FCU</p>

> ??? tip "Additional Information" 

Reviewer: separate collapsible "What are Flight Phases?"

    __Flight phases__<br/>
    The A380 uses flight phases to manage different parts of a flight. These are preflight, takeoff, climb, cruise, descent, approach, go around, done. They match the PERF pages in the MCDU (see [Preparing the MCDU](03_preparing-fms)).

>     - __Protections__<br/>

Reviewer: separate collapsible "What are Flight Envelope Protections?"

TODO: link to the separate page for protections

- ATC (Ground or Tower) has instructed us to hold at a runway holding point and wait until we are cleared to "line up" or "take off".<br/>
- Aircraft is still in TAXI state (see previous chapters) and parking brakes are set.

![Holding at runway holding point](../assets/beginner-guide/takeoff-climb-cruise/Holding.png "Aircraft holding at runway holding point"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot with 380</p>

Typically, it is here at the latest that we are asked to switch to Tower ATC frequency for takeoff clearance.

While approaching the runway holding point or at the latest at the runway
holding point the **"Before takeoff checklist"** needs to be completed.

!!! info "Before takeoff checklist"

>     ![Before Takeoff Checklist](../assets/beginner-guide/takeoff-climb-cruise/BeforeTakeOffChecklist.png "Before Takeoff Checklist"){ loading=lazy align=right width=30% } <p style="color:yellow; font-size:18px;">TODO: update checklist</p>

Reviewer: instead of the screenshot do it with the "white sop lines"

    The "Before Takeoff" checklist is divided into two parts: Down to the line, or Below the line.

??? tip ""The Line" Definitions"

>     - **"Down to the line"** (or "Above the line") means **before** "ATC Takeoff Clearance".

Reviewer: I would write this differently today:
explain the line is actually a line in the checklist for the current flight phase - and the flow of procedures is devided into above ("down to the line") and below the line

1. Check OVHD panel: APU off, no lights visible under normal circumstances<br/>
  (exception: Pack 1+2 might be OFF if part of procedure)

`OVHD PANEL .............................................CHECK DISCONNECTED`<br/>
`APU ...................................................................OFF`<br/>

> ![OVHD Panel at Lineup](../assets/beginner-guide/takeoff-climb-cruise/OVHDPanel.png "OVHD Panel at Lineup"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update OHP screenshot</p>

Reviewer: put this also in an admonition

```suggestion
??? note "OVHD Panel at Lineup"
    ![OVHD Panel at Lineup](../assets/beginner-guide/takeoff-climb-cruise/OVHDPanel.png "OVHD Panel at Lineup") 
    {loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update OHP screenshot</p>
```

`FLIGHT CONTROLS .....................................................CHECK`<br/>
`FLIGHT INSTRUMENTS ..................................................CHECK`<br/> 

> - Advise Cabin Crew

Reviewer: Make sure to use the A380 SOP:
![image](https://github.com/flybywiresim/docs/assets/16833201/7476f52b-5dca-44a4-b7f0-6eb6b0036228)

This would also be a SOP step and a note below

    ![Cabin notification](../assets/beginner-guide/takeoff-climb-cruise/CALLS-all.png "Cabin notification"){loading=lazy align=right width=45%} <p style="color:yellow; font-size:18px;">TODO: update screenshot with new CALL panel</p>

    - The cabin crew will notify the pilots with a "Cabin Ready" button once they are ready for takeoff. 
    - In the FBW A380, this is simulated by pressing the `CALLS ALL` button on the left of the overhead panel. 
    - This will set the "Cabin Ready" status as shown in the ECAM.

`SET FLAPS TO TAKEOFF SETTING ........................................CHECK`<br/>
`SET V SPEEDS ........................................................CHECK`<br/>

- Check Squawk

> ![ATC-TCAS](../assets/beginner-guide/takeoff-climb-cruise/ATC-TCAS.png){loading=lazy align=right width=45%}

Reviewer: Same here - SOP step and note below

`SQUAWK ID NUMBER ....................................................CHECK`<br/>
`SET SQUAWK .....................................................AUTO OR ON`<br/>
`SET ALT RPTG ...........................................................ON`<br/>
`COMM FREQUENCY ......................................................CHECK`<br/>

> - Check ECAM - no blue writing should be visible for these:

Reviewer: SOP step (with SOP step I mean this:

`SQUAWK ID NUMBER ....................................................CHECK`<br/>

`AUTO BRAKE ............................................................MAX`<br/>
`SIGN ...................................................................ON`<br/>
`CABIN ...............................................................READY`<br/>
`SPLRS .................................................................ARM`<br/>
`FLAPS ..................................................................TO`<br/>
`TO CONFIG ..........................................................NORMAL`<br/>

<p style="color:yellow; font-size:18px;">TODO: update screenshot with new ECAM</p>

> - Press T.O. Config button below the ECAM to check takeoff configuration

Reviewer: SOP Step

> - Check radar panel:

Reviewer: SOP Step

Shortly before we start our takeoff roll, we do the following steps:

`PACKS .......................................................CHECK AS REQ.`<br/>

>   (some airlines take off with Packs `OFF` to allow more power to thrust and save fuel - not necessarily required)

Reviewer: note admon
